nsis/visuald.nsi

```diff
@@ -330,7 +330,7 @@ ${MementoSection} "Install DMD" SecDMD
 
   ; not using ${File} to keep compiler even if Visual D uninstalled
 
-  !define DmdBaseDir "$CompilerInstallDir\dmd${DMD_VERSION}"
+  !define DmdBaseDir "$CompilerInstallDir\dmd-${DMD_VERSION}"

```

Comment:
This name is also used in the update process, so changes would have to be made in visuald/updates.d, too. I think it fits with the default D installation that uses `dmd2` in the same folder, though.

Reply:
It's ugly to append a "full" version number to the product name directly, so in most cases there would be a hyphen or underline to separate them, such as the name of dmd installer executive.